docs/contributor/01-00-installation.md

```diff
@@ -67,7 +67,7 @@ make docker-build
   </details>
 </div>
 
-6. Deploy API Gateway Operator.
+6. Deploy API Gateway Operator and `kyma-system` Namespace.
```

Comment:
```suggestion
6. Deploy API Gateway Operator in the `kyma-system` Namespace.
```

Reply:
Wait, that should have been "and" and "in"